pages/k8s/quickstart.md

```diff
@@ -0,0 +1,265 @@
+---
+title: "Quickstart | Canonical Distribution of Kubernetes&reg;"
+keywords: quickstart
+tags: [getting_started]
+sidebar: k8smain-sidebar
+permalink: quickstart.html
+layout: base
+toc: False
+summary: This page is currently a work in progress. For existing documentation, please visit <a href="https://kubernetes.io/docs/getting-started-guides/ubuntu/"> https://kubernetes.io/docs/getting-started-guides/ubuntu/ </a>
+---
+<style>
+img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 75%;
+  height: auto;
+}
+</style>
+
+<div class="p-content__row">
+    <div class="u-equal-height">
+        <div class="col-6">
+
+<h2> Quick Start</h2>
+
+<p>The Canonical Distribution of Kubernetes<sup>&reg;</sup> delivers a ‘pure K8s’
+experience, tested across a wide range of clouds and integrated with modern
+metrics and monitoring. It works across all major public clouds and private
+infrastructure, enabling your teams to operate Kubernetes clusters on demand,
+anywhere.</p>
+
+<p>With this quick start guide and some tools from Canonical, you can have all
+this distilled Kubernetes expertise operating in your own  cluster
+running on a cloud of your choice in minutes!</p>
+
+</div>
+<div class="col-6 u-align--right">
+    <img style="border: 0" src="./images/k8s-bundle.png" width="350" height="350">
+</div>
+</div></div>
+
+<hr class="is-deep">
+
+## What you'll need
+
+  *  An Ubuntu 18.04 LTS or 16.04 LTS environment to run the commands (or another operating system which supports `snapd` - see the [snapd documentation][snapd-docs])
+  * Account credentials for one of the following public clouds:
+      -  [Amazon Web Services][cloud-aws], including AWS-China and AWS-gov
+      -  [CloudSigma][cloud-cloudsigma]
+      - [ Google Cloud platform ][cloud-google]
+      - [Joyent][cloud-joyent]
+      - [Microsoft Azure][cloud-azure], including Azure-China
+      - [Oracle Cloud][cloud-oracle]
+      - [Rackspace][cloud-rackspace]
+
+
+<div class="p-notification--positive"><p class="p-notification__response">
+<span class="p-notification__status">Note:</span> If you don't meet these
+requirements, there are additional ways of installing the <emphasis>Canonical
+Distribution of Kubernetes<sup>&reg;</sup></emphasis>, inluding additional OS
+support and an entirely local deploy. Please see the more general
+<a href="./install"> Installing CDK</a> page for details. </p></div>
+
+## Install the tools
+
+The recommended way to install the **Canonical Distribution of
+Kubernetes&reg;** is with the tools <a href="https://jujucharms.com"> Juju</a>,
+an application modelling tool, and <a href="https://conjure-up.io">
+conjure-up</a>, a helper app for the guided install of complex  applications.
```

Comment:
"helper app" sounds a little off to me. How about "a guided installer for complex applications"?

Reply:
ok